### ISSUE TITLE
Cannot create a new simulation after recent updates

### ISSUE THREAD

**Issue by mrakitin** [radiasoft/sirepo#299]
When I try to create a new simulation (e.g., with the idealized undulator) in expert mode, I get the following error in the server log:
```python
10.0.2.2 - - [18/Aug/2016 15:10:00] "POST /new-simulation HTTP/1.1" 500 -
Traceback (most recent call last):
  File "/home/vagrant/.pyenv/versions/2.7.10/lib/python2.7/site-packages/flask/app.py", line 2000, in __call__
    return self.wsgi_app(environ, start_response)
  File "/home/vagrant/.pyenv/versions/2.7.10/lib/python2.7/site-packages/beaker/middleware.py", line 155, in __call__
    return self.wrap_app(environ, session_start_response)
  File "/home/vagrant/.pyenv/versions/2.7.10/lib/python2.7/site-packages/flask/app.py", line 1991, in wsgi_app
    response = self.make_response(self.handle_exception(e))
  File "/home/vagrant/.pyenv/versions/2.7.10/lib/python2.7/site-packages/flask/app.py", line 1567, in handle_exception
    reraise(exc_type, exc_value, tb)
  File "/home/vagrant/.pyenv/versions/2.7.10/lib/python2.7/site-packages/flask/app.py", line 1988, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/vagrant/.pyenv/versions/2.7.10/lib/python2.7/site-packages/flask/app.py", line 1641, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/home/vagrant/.pyenv/versions/2.7.10/lib/python2.7/site-packages/flask/app.py", line 1544, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/home/vagrant/.pyenv/versions/2.7.10/lib/python2.7/site-packages/flask/app.py", line 1639, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/vagrant/.pyenv/versions/2.7.10/lib/python2.7/site-packages/flask/app.py", line 1625, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/home/vagrant/src/radiasoft/sirepo/sirepo/server.py", line 304, in app_new_simulation
    new_simulation_data = _parse_data_input(validate=True)
  File "/home/vagrant/src/radiasoft/sirepo/sirepo/server.py", line 626, in _parse_data_input
    return simulation_db.fixup_old_data(data['simulationType'], data) if validate else data
  File "/home/vagrant/src/radiasoft/sirepo/sirepo/simulation_db.py", line 116, in fixup_old_data
    sirepo.template.import_module(simulation_type).fixup_old_data(data)
  File "/home/vagrant/src/radiasoft/sirepo/sirepo/template/srw.py", line 234, in fixup_old_data
    if data['models']['fluxReport'] and 'photonEnergyPointCount' not in data['models']['fluxReport']:
KeyError: 'models'
```

Rob, could you please look at it?

**Comment by robnagler** [comment]
Is this fixed @mrakitin ?

**Comment by mrakitin** [comment]
Yes, it's fixed in 4079b184a71171df25e7dc8736e59572e3b64276.
